includes/api/Cookies_Settings_Shortcode.php

```diff
@@ -0,0 +1,80 @@
+<?php
+// Shorcode which enables the form
+// ont the Cookies Settings page
+function shortcode_settings_page( $atts ) {
+    $a = shortcode_atts( array(
+    'usage' => 'show',
+    'settings' => 'hide',
+    'necessary' => 'show',
+    ), $atts );
+    
+    $accept_usage_option = "";
+    $accept_settings_option = "";
+    $reject_usage_option = "";
+    $reject_settings_option = "";
+
+    $cookies_policy_to_obj = decode_cookie('cookies_policy');
+
+    // If the user accept the Google Analitics cookies
+    // turn ON the radio selection
+    if ($cookies_policy_to_obj->usage === true) {
+        $accept_usage_option = "checked='checked'";
+    } else {
+        $accept_usage_option = "";
+    }
+
+    // If the user doesn't consent the cookies
+    // turn OFF the radio selection
+    if ($cookies_policy_to_obj->usage === false) {
+        $reject_usage_option = "checked='checked'";
+    } else {
+        $reject_usage_option = "";
+    }
+
+    // If the user accept the Settings cookies
+    // turn ON the radio selection
+    if ($cookies_policy_to_obj->settings === true) {
+        $accept_settings_option = "checked='checked'";
+    } else {
+        $accept_settings_option = "";
+    }
+
+    // If the user reject the Settings cookies
+    // turn OFF the radio selection
+    if ($cookies_policy_to_obj->settings === false) {
+        $reject_settings_option = "checked='checked'";
+    } else {
+        $reject_settings_option = "";
+    }
+
+    $output = '<form method="post" action="/latin/cookies/" id="ds-cookie-consent-form" class="tna-form tna-form-engagement">';
```

Comment:
This use of string concatenation with ternaries and Boolean logic is very difficult to follow. Would it be better to use a templating approach (Perhaps using Heredoc) https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc